cryodrgn/fft.py

```diff
@@ -3,7 +3,7 @@
 try:
     import cupy as cp
 except ImportError:
-    cp = None
+    cp = np
```

Comment:
Clever. Maybe add a comment explaining that this is for type checking?

Reply:
Well, in this case I wasn't thinking of this as being a clever trick just for the type-checker, but a more general claim that `cp` can be used anywhere in the code with the assumption that it supports the fft operations we're doing. I'm not a fan of `if we're using cupy then do this otherwise do that` as we're currently doing in the `fft` module (I had to merge it in because it was a useful community contribution at the time). `cupy`/`numpy` etc. go through great lengths to make sure their APIs are compatible and we should take advantage of that I think..